src/dxtbx/nexus/__init__.py

```diff
@@ -6,11 +6,13 @@
 import h5py
 import numpy as np
 
+import cctbx
 from cctbx import eltbx
 from scitbx.array_family import flex
 
 import dxtbx.model
 from dxtbx import flumpy
+from dxtbx.format.nexus import convert_units
```

Comment:
I would prefer to avoid dependencies on the "old" `dxtbx.format.nexus` code if at all possible. We've been using [`pint`](https://pint.readthedocs.io/en/stable/) for unit conversions in the new NXmx code rather than custom unit conversion code as before.